### ISSUE TITLE
Render Style does not change from Bulk Edit on 'Search Effects' screen. 

### ISSUE THREAD

**Issue by atticusm** [xLightsSequencer/xLights#3644]
NOTE: IF YOU DO NOT ATTACH A SEQUENCE THAT DEMONSTRATES YOUR PROBLEM THEN THERE IS A HIGH PROBABILITY YOUR ISSUE WILL JUST BE CLOSED AND NOT INVESTIGATED.

**Describe the bug**

**To Reproduce**
(and/or link to a short video showing the problem with audio track describing what you are doing)
1. Go to View
2. Windows
3. Search Effects
4. Search for "Per Preview"
5. Select multiple group options holding down control
6. Right-click on Render Style and select Bulk Edit
7. Select Default  
8. Hit OK

**Expected behavior**
All the effects selected in the search panel should now be bulk edited to the render style

**Screenshots**

**Versions (please complete the following information):**
 - OS: Windows 10 x64
 - xLights version 2022.25 64bit wxWidgets 3.3.0

**Attachments**
I'm attaching screenshots of the process. In the first one you can see I have multiple groups (2) selected. Right-click on Render Style to choose Bulk Edit. Choose Default and hit ok. Next picture if when I refresh the panel and still see the same groups with the old render style. 
![RenderStyleDidNotChange](https://user-images.githubusercontent.com/5733682/208323647-7e344e83-80c2-4903-b2aa-20b4dd162f0c.jpg)
![ChangingToDefault](https://user-images.githubusercontent.com/5733682/208323648-e8bc7f84-ee38-4aa4-a6db-7ca86d27e211.jpg)
![FirstPic](https://user-images.githubusercontent.com/5733682/208323649-8e1b9803-b5f5-45b1-a8c8-117fc12bd9d7.jpg)

I would rather not publically upload the sequence since it's from a vendor. I am happy to package it and send to a developer once requested. 

Thank you for everything.

**Comment by derwin12** [comment]
Atticus - are you sure it is not working .. it works for me ...  v.22
--Daryl

**Comment by atticusm** [comment]
Howdy Daryl. Happy New Years. Sorry for the long delay. I tested it again and it appears that *some* effects will update but not others. For example, I selected 6 effects, right-clicked on render style, selected Default and hit ok, When I reran the search it applied the change to only 2 out of those 6? 

I also upgraded to [2023.01]
![SelectEffectAfter](https://user-images.githubusercontent.com/5733682/211635801-4bc20e6a-bd83-4db1-b7b8-174cb8599583.jpg)

https://user-images.githubusercontent.com/5733682/211635803-68c79572-c4b7-4b7f-a879-995ebb4a6cb9.mp4


https://user-images.githubusercontent.com/5733682/211635812-466a36d9-ea4a-46cf-b638-cf793c9b68ca.mp4
